vyper/signatures/function_signature.py

```diff
@@ -168,7 +168,8 @@ def from_definition(cls,
 
         # Validate default values.
         for default_value in getattr(code.args, 'defaults', []):
-            if not isinstance(default_value, (ast.Num, ast.Str, ast.Bytes, ast.List)):
+            allowed_types = (ast.Num, ast.Str, ast.Bytes, ast.List, ast.NameConstant)
```

Comment:
I think we have is_literal on the types directly now, wold be properly clearer to use that ;)